docs/galchsa-bio.md

```diff
@@ -0,0 +1,7 @@
+# GitHub Student
+
+**Name:** GitHub Student
+
+**Occupation:** Engineer
+
+**Location:** Near elevator. 
```

Comment:
Ha! :smiley_cat: 

What if we removed the period here to make it like the other lines?